src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -2484,7 +2491,90 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  '/nodes/{nodeId}/calculateSize':
+    post:
+      tags:
+        - nodes
+      summary: Calculate a folder size
```

Comment:
Are we supporting the API to calculate the size of a file. If yes, then we shd mention in the description.

Reply:
We are only calculating the size of a folder not file.